appinventor/components/src/com/rit/appinventor/components/runtime/ros_bridge.java

```diff
@@ -7,21 +7,25 @@
 import com.google.appinventor.components.annotations.SimpleFunction;
 import com.google.appinventor.components.annotations.SimpleObject;
 import com.google.appinventor.components.common.ComponentCategory;
-import com.google.appinventor.components.common.YaVersion;
 import com.google.appinventor.components.runtime.AndroidNonvisibleComponent;
 import com.google.appinventor.components.runtime.Deleteable;
 import com.google.appinventor.components.runtime.OnResumeListener;
 import com.google.appinventor.components.runtime.OnStopListener;
+import com.google.appinventor.components.runtime.ComponentContainer;
 
 @RequiresApi(api = Build.VERSION_CODES.CUPCAKE)
-@DesignerComponent(version = YaVersion.ROSBRIDGE_COMPONENT_VERSION,
+@DesignerComponent(version = 1,
```

Comment:
Should this stay as 1?

Reply:
I changed it because the YaVersion file had no ROSBRIDGE_COMPONENT_VERSION variable. It wasnt recognizing it and so I was getting errors that prevented me from building

Reply:
Weird, we can probably look into this more at the next meeting.